mock/service/controller.go

```diff
@@ -341,3 +341,23 @@ func (s *service) ControllerGetCapabilities(
 		},
 	}, nil
 }
+
+func (s *service) CreateSnapshot(ctx context.Context,
+	req *csi.CreateSnapshotRequest) (*csi.CreateSnapshotResponse, error) {
+	return &csi.CreateSnapshotResponse{}, nil
```

Comment:
Do you want to return an error for now ("not implemented") until we can return a real mock object (like `CreateVolume(...)` does above)?

Reply:
Same for `DeleteSnapshot` and `ListSnapshot` below?

Reply:
make sense, will do for both